frameworks/CSharp/beetlex/Benchmarks/Program.cs

```diff
@@ -22,21 +30,21 @@ public static void Main(string[] args)
         public object plaintext(IHttpContext context)
         {
             context.Response.Header[HeaderTypeFactory.DATE] = DateTime.Now.ToUniversalTime().ToString("r");
-            return new TextResult("Hello, World!");
+            return plaintextResult;
         }
 
         public object json(IHttpContext context)
         {
             context.Response.Header[HeaderTypeFactory.DATE] = DateTime.Now.ToUniversalTime().ToString("r");
-            return new JsonResult(new JsonMessage { message = "Hello, World!" });
+            return new JsonResult(jsonMessage);
```

Comment:
The jsonMessage has to be instantiated on every request per the test rules. Please revert this change.